### DISCUSSION TITLE
Missing auth in v2

### DISCUSSION THREAD

**Discussion by sbe-arg** [aceberg/WatchYourLAN#129] [Q&A] [answered]
How do we enable auth in v2?

**Comment by aceberg** [comment]
There is no auth in v2 right now.

1. v1 has session cookie based auth, which is not the best solution. I'm thinking about JWT-token auth in v2, but it won't be very soon.

2. If you need to expose WYL to an unsafe network, you can use it with a reverse proxy that supports auth or SSO (Authelia should work).

**Comment by aceberg** [comment, marked as answer]
Hi! There is an update about Auth. I made a separate app [ForAuth](https://github.com/aceberg/ForAuth) to use in all my apps. JWT will probably come later, now it supports session-cookie.
Here is an example, how to use it with WYL:
https://github.com/aceberg/WatchYourLAN/blob/main/docker-compose-auth.yml
